mmv1/products/apigee/EnvironmentKeyvaluemap.yaml

```
@@ -0,0 +1,74 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'EnvironmentKeyvaluemaps'
+description: |
+  Collection of key/value string pairs.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Using key value maps': 'https://cloud.google.com/apigee/docs/api-platform/cache/key-value-maps'
+  api: 'https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.keyvaluemaps/create'
+
+base_url: '{{env_id}}/keyvaluemaps'
+create_url: '{{env_id}}/keyvaluemaps'
+delete_url: '{{env_id}}/keyvaluemaps/{{name}}'
+self_link: '{{env_id}}/keyvaluemaps/{{name}}/entries'
+import_format: ['{{env_id}}/keyvaluemaps/{{name}}', '{{env_id}}/{{name}}']
+skip_sweeper: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'apigee_environment_keyvaluemaps_basic'
+    primary_resource_id: 'apigee_environment_keyvaluemaps'
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+    skip_test: true
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'apigee_environment_keyvaluemaps_test'
+    primary_resource_id: 'apigee_environment_keyvaluemaps'
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+    skip_docs: true
```

```suggestion
    skip_docs: true
    min_version:
      beta
```

I think we need to add this flag so that it will only generate tests in the beta provider, since we have some beta-only resources in the test.

---

Added. Thanks